frontends/web/src/components/update/update.tsx

```diff
@@ -41,7 +41,3 @@ const Update = ({ file }: TProps) => {
     </Status>
   );
 };
-
-const HOC = load<TProps>({ file: 'update' })(Update);
```

Comment:
fyi: this would fetch update and write the result into a prop 'file'.
https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/web/src/decorators/load.tsx

there is a few more around which I'll try to refactor soon. load also accepts a function instead, for example here: 
https://github.com/digitalbitbox/bitbox-wallet-app/pull/2116

Reply:
> fyi: this would fetch update and write the result into a prop 'file'.

is this file somewhere in appfolder.dev or what is this file?

Reply:
It's just a JSON on our servers, see the go backend endpoint code https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/backend/update.go

Reply:
Oh, the 'update' endpoint will do that. I thought you were talking about the load HoC :)

Reply:
ah, no file is just a string in which prop to put the result in.